internal/cmd/commands/server/listener_reload_test.go

```diff
@@ -83,7 +83,6 @@ listener "tcp" {
 `
 
 func TestServer_ReloadListener(t *testing.T) {
-	t.Parallel()
```

Comment:
Could you add a comment with a warning that this test cannot be run in parallel (and why) so we don't reintroduce this accidentally?

Reply:
That actually I will revert...it was left over from trying to figure out one of the data races.